usr/sbin/pkcsslotd/socket_server.c

```diff
@@ -278,10 +289,20 @@ static int client_socket_init(int socket, int (* xfer_complete)(void *client),
                 "%d (%s).", __func__, socket, err, strerror(err));
         return -err;
     }
-
+#if defined(_AIX)
+    struct poll_ctl_ext evt = {
```

Comment:
please define this variable above at the beginning of the lock (i.e. in the else case if the if !defined(AIX).